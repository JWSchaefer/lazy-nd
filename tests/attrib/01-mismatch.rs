use lazy_nd::lazy_nd;

#[lazy_nd(dim = G)]
struct Test<const D: usize> {}
fn main() {}
