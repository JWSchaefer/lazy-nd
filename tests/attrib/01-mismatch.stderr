error: Expected generic `<const G : usize>`
 --> $DIR/01-mismatch.rs:4:12
  |
4 | struct Test<const D: usize> {}
  |            ^
